deploy/helm/cht-chart/templates/couchdb-claim0-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and (not .Values.couchdb.clusteredCouch_enabled) (eq .Values.environment "remote") }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    cht.service: couchdb-claim0
+  name: couchdb-claim0
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ .Values.couchdb.couchdb_node_storage_size }}
```

Comment:
I think this should be `.Values.couchdb.single_node_storageSize`

Reply:
that variable has been removed in favor of this one variable that sits under the `couchdb` property.